addons/googledrive/tests/factories.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""Factory boy factories for the Google Drive addon."""
+import datetime
+import factory
+
+from dateutil.relativedelta import relativedelta
+
+from factory.django import DjangoModelFactory
+from osf_tests.factories import UserFactory, ProjectFactory, ExternalAccountFactory
+
+from addons.googledrive.models import NodeSettings
+from addons.googledrive.models import UserSettings
+
+
+class GoogleDriveAccountFactory(ExternalAccountFactory):
+    provider = 'googledrive'
+    provider_id = factory.Sequence(lambda n: 'id-{0}'.format(n))
+    oauth_key = factory.Sequence(lambda n: 'key-{0}'.format(n))
+    oauth_secret = factory.Sequence(lambda n: 'secret-{0}'.format(n))
+    expires_at = datetime.datetime.now() + relativedelta(days=1)
```

Comment:
https://docs.djangoproject.com/en/1.10/topics/i18n/timezones/#naive-and-aware-datetime-objects